workspace/assets/js/dev/panel.js

```diff
@@ -161,7 +161,7 @@
 		var attached = false;
 
 		$(document).on('keydown', function (e) {
-			if (e.which === global.keys.escape) {
+			if (e.which === 27) {
```

Comment:
This should use the new keys modules.

Reply:
Yeah if we import the module only in dev. I don't think it's worth it to import a module on a prod build only to be used in an dev tool.

Reply:
@PascalPiche wdyt ?

Reply:
Let's wait for @PascalPiche, but it should at least be a variable so it would be === escape

Reply:
convert to a variable for now. but yep. we should split the build to make a css and js extra part for dev / logged in user.

Reply:
I have a full day soon (ishe) to work on the framwork. I'll keep up posted

Reply:
@fhamon :+1: